### ISSUE TITLE
Unsupported CUP files for POI

### ISSUE THREAD

**Issue by scls19fr** [tumic0/GPXSee#219]
Hello,

I'm trying to import a CUP file as POI (because map doesn't show enough details) but GPXSee doesn't seem to support it

See such a `.cup` file
http://download.xcsoar.org/waypoints/France.cup

```
$ head France.cup
"Abbeville Gld",,FR,5008.583N,00149.900E,67.0m,5,020,1260.0m,,"Flugplatz"
"Agen La Garenne",,FR,4410.483N,00035.433E,61.0m,5,110,2170.0m,121.300,"Flugplatz"
"Aigrefeuille Ul",,FR,4607.633N,00057.117W,30.0m,3,100,350.0m,,"Landefeld"
"Ainy Le Vieil",,FR,4640.400N,00232.050E,199.0m,3,100,600.0m,,"Landefeld"
"Aire Lys Ulm ?",,FR,5037.217N,00225.083E,35.0m,3,,,,"Landefeld"
"Aire Sur L Adour",,FR,4342.483N,00014.817W,80.0m,5,120,980.0m,118.975,"Flugplatz"
"Aires Aero",,FR,4354.450N,00002.883E,160.0m,3,070,350.0m,123.500,"Landefeld"
"Aiton Champs",,FR,4533.417N,00613.667E,300.0m,3,110,300.0m,,"Landefeld"
"AiX En Diois Cls",,FR,4441.350N,00524.533E,531.0m,2,150,300.0m,130.000,"Flugplatz"
"AiX Les Milles",,FR,4330.317N,00522.050E,105.0m,5,150,1590.0m,118.750,"Flugplatz"
$ tail France.cup
"Westpunkt Fur Dr",,FR,4500.650N,00512.483E,388.0m,1,,,,"FURDREIE"
"Wingen Gare",,FR,4855.217N,00722.850E,220.0m,1,,,,"BAHNHOF"
"Winnezeele A25 A",,FR,5050.250N,00234.367E,30.0m,17,,,,"A25AB014"
"Wizern",,FR,5042.217N,00212.483E,44.0m,1,,,,
"Wp West Virtuell",,FR,4453.583N,00512.350E,1084.0m,1,,,,"VIRTUELL"
"YchouX",,FR,4419.800N,00057.700W,33.0m,1,,,,
"Ygos St Saturnin",,FR,4358.717N,00044.300W,72.0m,1,,,,"BAHNHOF"
"Yorre Saint",,FR,4603.900N,00327.683E,293.0m,1,,,,"SAINT"
"YssingeauX Centr",,FR,4508.567N,00407.350E,886.0m,1,,,,"CENTRE VI"
"Zone Derogatoire",,FR,4650.167N,00049.800W,165.0m,1,,,,"DEROGATOI"
```

And file format description
http://download.naviter.com/docs/CUP-file-format-description.pdf

Kind regards

**Comment by tumic0** [comment]
.cup files are not supported at the moment. Makes sense to add support for them, but this will have to be a little bit more complex work as doing so will require to implement some kind of a more generic CSV reader so other formats based on CSV can be easily added without code duplication.

**Comment by tumic0** [comment]
Implemented, will be part of 7.12.
